### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#344] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325367-b3b53f95-d2cf-4db4-84be-6c7c46ba2fd2.png)

**Comment by i0nut02** [comment]
<img width="616" alt="Schermata 2022-02-05 alle 12 08 23" src="https://user-images.githubusercontent.com/99051485/152639605-f0e3746b-493f-4cd0-8fc5-b53834ff985f.png">

**Comment by Exyss** [comment]
![Boh(23)](https://user-images.githubusercontent.com/65043207/152687219-3cfdb688-5716-482e-a6ec-cf4df31a5ad2.jpg)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217814300-55d0ced0-c3b9-4ae5-9121-eb6f7787f657.png)
